### DISCUSSION TITLE
Target mod should have a multiplier of 0.1

### DISCUSSION THREAD

**Discussion by Doksha** [ppy/osu#21012] [Ideas] [answered]
This is probably already planned, but I cant seem to find anything on it. Target is basically just a bad relax/autopilot hybrid, it makes most maps 10x easier, with the only downside being you have to FC to get a score (without nofail) and its just absurdly annoying to play.
Currently it has no multiplier, so you can just go on 10* maps and FC them with target to get an easy first place. Lazer even accounts for target when calculating star rating, so it should also apply to score

**Comment by ghost** [comment]
i say something like 0.5. it shouldn't be nerfed to the ground (it still has some difficulty, especially near the end) but it definitely needs some sort of reduction to prevent it from appearing in leaderboards

**Reply by Doksha** [reply]
no, 0.5 is way to big, mod multipliers should apply to the worse case scenario, you shouldnt get only 0.56x score for DTing a 10* with target since even NM players would have an insane disadvantage (literally everyone playing without target would have a huge disadvantage). Even 0.3x would be quite a lot.
What target does is it hugely reduces aim complexity and hugely reduces tapping complexity (at least for any map above 5*) with its only difficulty being starting after breaks.
Relax and autopilot at least have complexity to the skillset opposite to what they make easier.
Target basically has nothing to do with the map itself.

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/pull/21054
